.vscode/launch.json

```diff
@@ -0,0 +1,61 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Main Process",
+      "cwd": "${workspaceRoot}",
+      "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron",
+      "runtimeArgs": [
+        "src",
+        "--disable-dev-mode"
+      ],
+      "windows": {
+        "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron.cmd"
+      },
+      "program": "${workspaceRoot}/src/main.js",
+      "outFiles": [
+        "${workspaceRoot}/src/main_bundle.js"
+      ],
+      "sourceMaps": true,
+      "preLaunchTask": "Build sources"
+    },
+    /*
```

Comment:
this is ok?

Reply:
@cpanato Yes, because while the config seems correct @yuya-oc hasn't succeeded to make it work from his side.

```
Sadly renderer process debugging is not working for me.
```